bots/pr/src/main/java/org/openjdk/skara/bots/pr/PullRequestCommandWorkItem.java

```diff
@@ -228,4 +228,22 @@ public String toString() {
     public String workItemName() {
         return "pr-command";
     }
+
+    /**
+     * This method returns all the comments in the pr including comments in reviews(review body) and file specific comments
+     */
+    private List<Comment> getAllComments() {
+        // Regular comments
+        List<Comment> comments = new ArrayList<>(prComments());
+        // Comments in reviews
+        comments.addAll(pr.reviews().stream()
+                .map(review -> new Comment("Review" + review.id(), review.body().orElse(""), review.reviewer(), review.createdAt(), null))
+                .toList());
+        // File specific comments
+//        comments.addAll(pr.reviewComments().stream()
+//                .map(reviewComment -> new Comment("ReviewComment" + reviewComment.id(), reviewComment.body(), reviewComment.author(), reviewComment.createdAt(), reviewComment.updatedAt()))
+//                .toList());
```

Comment:
Since we have decided not to do this, I recommend to remove this commented out code.

Reply:
Sure